### PR TITLE
Log level constants

### PR DESCRIPTION
**Pull request by mangelajo**

### CODE REVIEW

**Comment by mangelajo** [comment]
ooops
mixed up commit, sorry

File: pkg/log/loglevels.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package log
+
+const (
+	TRACE_LEVEL   = 100
+	DEBUG_LEVEL   = 10
+	VERBOSE_LEVEL = 4
+	INFO_LEVEL    = 0
```
Comment (mangelajo) [inline pkg/log/loglevels.go]: Sorry I think I got it wrong, and default should be 1?

Comment (tpantelis) [inline pkg/log/loglevels.go]: I don't think we need INFO_LEVEL - one would just use `klog.Infof(...)`.

Comment (mangelajo) [reply pkg/log/loglevels.go]: Probably we should remove "INFO_LEVEL" altogether and then, use
klog.Info( ... ) for "always-on" logs.

File: pkg/log/loglevels.go [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package log
+
+const (
+	TRACE_LEVEL   = 100
+	DEBUG_LEVEL   = 10
+	VERBOSE_LEVEL = 4
```
Comment (tpantelis) [inline pkg/log/loglevels.go]: What does VERBOSE_LEVEL mean? 

I was thinking:

TRACE = 3
DEBUG = 2

Comment (mangelajo) [reply pkg/log/loglevels.go]: sounds good

File: pkg/log/loglevels.go [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package log
+
+// Log levels defined for use with:
+//   klog.V(__).Info
+
+const (
+	// INFO_LEVEL : This level is for anything which does not happen very often, and while not being
+	//              an error, needs to be on the logs, because it's important information:
```
Comment (tpantelis) [inline pkg/log/loglevels.go]:
```suggestion
	//              an error, is important information and helpful to be in the logs, eg:
```

File: pkg/log/loglevels.go [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+	//                * startup information
+	//                * HA failovers
+	//                * re-connections/disconnections
+	//                * ...
+	//              This level is not defined because you must use klog.Info helpers
```
Comment (tpantelis) [inline pkg/log/loglevels.go]:
```suggestion
	//              This level is not specifically defined as you would use the klog.Info helpers.
```

File: pkg/log/loglevels.go [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+	//
+	// DEBUG_LEVEL : should be used to provide logs on often occurring events but, which could be helpful
```
Comment (tpantelis) [inline pkg/log/loglevels.go]:
```suggestion
	// DEBUG_LEVEL : used to provide logs for often occurring events that could be helpful
```

File: pkg/log/loglevels.go [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+	//               for debugging errors.
+	DEBUG_LEVEL = 2
+	// TRACE_LEVEL : must be used for debug level errors dumping lots of information which generally would
```
Comment (tpantelis) [inline pkg/log/loglevels.go]:
```suggestion
	// TRACE_LEVEL : used for logging that occurs often or may dump a lot of information which generally 
        //               would be less useful for debugging but can be useful in some cases, for example tracing
	//               function entry/exit, parameters, structures, etc..
```

File: pkg/log/loglevels.go [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+	//              an error, needs to be on the logs, because it's important information:
+	//                * startup information
+	//                * HA failovers
+	//                * re-connections/disconnections
+	//                * ...
+	//              This level is not defined because you must use klog.Info helpers
+	//
+	// DEBUG_LEVEL : should be used to provide logs on often occurring events but, which could be helpful
+	//               for debugging errors.
+	DEBUG_LEVEL = 2
```
Comment (tpantelis) [inline pkg/log/loglevels.go]: Do you think we can omit the "_LEVEL" part to reduce real estate? I think `klog.V(log.DEBUG).Infof(...)` reads clearly.

Comment (sridhargaddam) [reply pkg/log/loglevels.go]: Big +1 to this.

Comment (mangelajo) [reply pkg/log/loglevels.go]: I like that, will do it.

**Comment by mangelajo** [comment]
Talking to tpantelis on whatsapp, this should be fine to merge, he's on the road and PTO.

**Pull request merged**